ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__4.11-aws-ovn.yaml

```diff
@@ -7,20 +7,20 @@ base_images:
     name: "4.11"
     namespace: ocp
     tag: cli
-  openshift_eventing-kafka-broker-src_knative-v1.6:
+  openshift_eventing-kafka-broker-src_knative-v1.7:
     name: eventing-kafka-broker-src
     namespace: openshift
-    tag: knative-v1.6
-  openshift_knative-eventing-src_knative-v1.6:
+    tag: knative-v1.7
+  openshift_knative-eventing-src_knative-v1.7:
     name: knative-eventing-src
```

Comment:
It seems `knative-eventing-src` was replaced with `eventing-src` by this commit https://github.com/openshift/release/commit/9e7f88f58d31608b63ff62933ec840431423c28c#diff-904a63d931dc06dc6f80feaeb5fd08bec7a9e22bf75eb81c652b34851fada744L204

@matzew @pierDipi Could you give your advice what we should change :sweat_drops:

Reply:
I think these images are not yet on the registry at all